StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -157,5 +169,16 @@ public static void FixMonsterSlayerQuest_IncludeReleaseofGoals_Postfix()
                 return;
             }
         }
+
+        public static string AuroraVineyardMatchesBundles()
+        {
+            
+            var bundlesPrice = _archipelago.SlotData.BundlePrice;
+            if (bundlesPrice.HasFlag(BundlePrice.Normal))
```

Comment:
Why not make an entry for BundlePrice Normal in the dictionary, and then rely on it always? Why append "Apples" to it, instead of having it built-in the dictionary value?